resources/hiding_ci/build_and_install_kernel.sh

```diff
@@ -2,26 +2,49 @@
 # Copyright 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-# We need sudo privilleges to install the kernel
-if [ "$(id -u)" -ne 0 ]; then
-  echo "This script must be run as root or with sudo privileges"
-  exit 1
-fi
-
-# Currently this script only works on Ubuntu instances
-if ! grep -qi 'ubuntu' /etc/os-release; then
-  echo "This script currently only works on Ubuntu."
-  exit 1
-fi
+# fail if we encounter an error, uninitialized variable or a pipe breaks
```

Comment:
is there a reason why we don't squash "ci: Allow build script to run without install"  commit into the previous one?

Reply:
Squashed the two together now